source_gen/lib/builders.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:build/build.dart';
+import 'package:glob/glob.dart';
+import 'src/builder.dart';
+
+const _outputExtensions = '.g.dart';
+const _partFiles = '.g.part';
+
```

Comment:
we'll eventually need a `BuilderFactory` here too. Depending on how you want to structure the tests you could make the class private or skip the `export` for it since all that's needed to be public is the factory.